packages/tally/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rabbitholegg/questdk-plugin-tally",
-  "version": "1.0.0-alpha.27",
+  "version": "1.0.0-alpha.28",
```

Comment:
Same comment as the CHANELOG - please see the readme

Reply:
fixed, plz, confirm